lib/bridge/MatrixHandler.js

```diff
@@ -451,6 +451,28 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
         yield this.ircBridge.sendMatrixAction(adminRoom, botUser, notice, req);
         return;
     }
+    else if (cmd === "!storepass") {
+        let domain = ircServer.domain;
+        let userId = event.user_id;
+        let notice;
+
+        try {
+            // Allow passwords with spaces
+            let pass = args.join('');
+            yield this.ircBridge.getStore().storePass(userId, domain, pass);
+            notice = new MatrixAction(
+                "notice", `Successfully stored password`
```

Comment:
We should mention that this only applies on *reconnects* and that they still need to poke NickServ initially. Can we name the domain as well please (since the server is often implicit, they may not have typed it in).

> Successfully stored password on ${domain}. When you next reconnect to ${domain}, this password will be automatically sent in a PASS command which most IRC networks will use as your NickServ password. This means you will not need to talk to NickServ. This does NOT apply to your currently active connection: you still need to talk to NickServ one last time to authenticate your current connection if you haven't already.